backend/main.py

```diff
@@ -0,0 +1,99 @@
+import sqlite3, uuid, json
+from bottle import Bottle, run, request, response
+
+app = Bottle()
+
+# DB mit SQLite
+con = sqlite3.connect("users.db")
+cur = con.cursor()
+
+def init():
+    createDB()
+
+def createDB():
+    sql_query = """
+    CREATE TABLE IF NOT EXISTS users (
+    id VARCHAR(36),
+    nick VARCHAR(30),
+    sessionid VARCHAR(30)
+    );"""
+    cur.execute(sql_query)
+    con.commit()
+
+def insertUserToDB(nick,sessionid):
+    sql_query = """
+    INSERT INTO users (id,nick,sessionid)
+    VALUES ('"""+str(uuid.uuid4())+"','"+nick+"','"+sessionid+"""');"""
```

Comment:
This looks like a SQL-Injection waiting to happen. Its probably better to use parameter substitution here.

Reply:
yes, very bad style. adjusted in the new commit using a parameterized sql request.

Reply:
The changes you performed result in the same string interpolation, just with nicer syntax. Take a look at the introduction in https://docs.python.org/3/library/sqlite3.html

For example the code might look like this:

```python
cur.execute(
    """
    INSERT INTO users (id, nick, sessionid)
    VALUES (?, ?, ?)
    """,
    (str(uuid.uuid4()), nick, sessionid)
)
```

There is also a named style if you prefer, directly quoting from the python docs:

```python
cur.execute("select * from lang where first_appeared=:year", {"year": 1972})
```